cg/store/filters/status_ordertype_application_filters.py

```diff
@@ -0,0 +1,33 @@
+from enum import Enum
+
+from sqlalchemy.orm import Query
+
+from cg.store.models import OrderTypeApplication
+
+
+def filter_applications_by_order_type(
+    ordertype_applications: Query, order_type: str, **kwargs
+) -> Query:
+    """Return application by order type."""
+    return ordertype_applications.filter(OrderTypeApplication.order_type.is_(order_type))
+
+
+def apply_ordertype_application_filter(
+    filter_functions: list[callable],
+    ordertype_applications: Query,
+    order_type: str = None,
```

Comment:
```suggestion
    order_type: OrderType = None,
```
There is an enum which is suitable for this

Reply:
fixed